braid/_braid.c

```diff
@@ -555,9 +554,15 @@ _braid_AdjointFeatureCheck(braid_Core core)
  * Macros used below
  *----------------------------------------------------------------------------*/
 
+#define _braid_SendIndexOff -2
```

Comment:
I initially thought that "Off" here referred to offest.  We could call this variable
_braid_SendIndexNull
or something like that.  It's a very minor thing...just a thought.

Reply:
I like that.  I'll make the change.